src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -209,6 +211,21 @@ export function ConfigEditor(props: Props) {
         label={selectors.components.ConfigEditor.Database.input}
         data-testid={selectors.components.ConfigEditor.Database.testID}
       />
+      <InlineField label="Send events to Amazon EventBridge" labelWidth={28}>
```

Comment:
If you add `style={{ alignItems: 'center' }}` the switch will be vertically aligned